server/guiserver/apps.py

```diff
@@ -106,6 +106,7 @@ def server():
         'jujugui.uuid': options.uuid,
         'jujugui.interactive_login': options.interactivelogin,
         'jujugui.gzip': options.gzip,
+        'jujugui.GTM_enabled': options.gtm,
```

Comment:
I find having all of 'gtm-enabled', 'gtm', and 'GTM_enabled' really confusing.  The horse is out of the barn, but I'd really prefer 'gtm-enabled' in config.yaml and 'gtm_enabled' everywhere else. If that is too pedantic can we kill 'GTM_enabled' in favor of just 'gtm'?